res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1394,27 +1418,31 @@ PioneerDDJSX.tempoResetButton = function(channel, control, value, status, group)
 PioneerDDJSX.autoLoopButton = function(channel, control, value, status, group) {
     if (value) {
         if (engine.getValue(group, "loop_enabled")) {
-            engine.setValue(group, "reloop_exit", true);
+            engine.setValue(group, "reloop_toggle", true);
         } else {
-            engine.setValue(group, "beatloop_" + 4 + "_toggle", true);
+            engine.setValue(group, "beatloop_activate", true);
```

Comment:
It's confusing that there are multiple buttons for reloop_toggle. However, it would also be redundant to use shift and this button for beatlooproll_activate because of the ROLL pad mode, so I'm not sure it matters much either way...?